.github/workflows/automerge.yml

```diff
@@ -0,0 +1,28 @@
+name: automerge
+on:
+  pull_request:
+    types:
```

Comment:
Is this list comprehensive? Wanted to ask about why these types are picked?

Reply:
I think it is comprehensive -- these were the recommended types for the standard configuration of this plugin, I think: https://github.com/broadinstitute/automerge-action#usage

but your comment is prompting me to look into this in more detail. I'm like 80% sure about this.